aibolit/utils/utils.py

```diff
@@ -0,0 +1,11 @@
+import re
+
+
+def remove_comments(string):
```

Comment:
@lyriccoder We shouldn't use utility classes. How bout creating a decorator which remove the comments of a string or file?